package.json

```diff
@@ -44,16 +44,16 @@
     "js-cookie": "^2.2.0",
     "msgpack-lite": "^0.1.26",
     "mustache-express": "^1.2.5",
-    "pako": "^1.0.10",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
+    "pako": "^2.0.4",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
     "reconnecting-websocket": "^4.4.0",
     "request": "^2.88.0",
     "request-promise-native": "^1.0.7",
     "semver": "^7.1.0",
     "stats-js": "^1.0.0",
     "styled-components": "^5.1.0",
-    "three": "^0.123.0",
+    "three": "^0.132.0",
```

Comment:
It looks like Three.js has breaking changes (which we get warnings about when building the client), causing the editor to break:
![image](https://user-images.githubusercontent.com/30075631/134057474-135d2f31-c7b8-471b-9c5c-650a34e932cc.png)